HW-2025-07-07/index.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Start</title>
+</head>
+<body>
+<h1>Volodymyr Mishchuk</h1>
+
+<h3>Persönliche Daten</h3>
+<p>+49 151 152 171 13</p>
+<p>06217 Merseburg</p>
+<p>Ruhrweg 8</p>
+<p>mishchuk.mail@gmail.com</p>
+<h3>Geburtsdatum</h3>
+<p>09.03.1991</p>
+<h3>Geburtsort</h3>
+<p>Narva-Jõesuu, Estland</p>
+<h3>Nationalität</h3>
+<p>Ukrainer</p>
+<h3>Nationalität</h3>
+<p>Verheiratet, 2 Kinder</p>
+
+<h3>Berufliche Erfahrung</h3>
+<ul>
+    <li>
+        02.2025 bis heute Full-Stack Developer Weiterbildung im IT-Bereich
+    </li>
+    <li>
+        04.2024-09.2024 Kurs B2 Beruf (Language Coach Institute - Leipzig)
+    </li>
+    <li>
+        10.2023-02.2024 Mitarbeiter Supermarkt Netto (Aushilfer)
+    </li>
+    <li>
+        10.2023-11.2023 Orientierungskurs (Dama.go Leipzig)
+    </li>
+    <li>
+        03.2023-10.2023 Kurs B1 Integationskurs (Dama.go Leipzig)
+    </li>
+    <li>
+        09.2022-03.2023 Kurs A1 Integationskurs (BVU Merseburg)
+    </li>
+    <li>
+        03.2022-09.2022 Erverbslos
+    </li>
+    <li>
+        2020-2022 Entwicklung und technische Wartung des Onlineshops
+    </li>
+    <ul>
+        <li>
+            Erstellung einer Website im Baukasten-Modus
+        </li>
+        <li>
+            Arbeit mit Zahlungssystemen
+        </li>
+        <li>
+            Moderation von Nutzern
+        </li>
+        <li>
+            Einrichtung automatischer E-Mail-Verteiler
+        </li>
+        <li>
+            Organisation und technische Betreuung von Online-Übertragungen
+        </li>
+        <li>
+            Bereitstellung von Multimedia-Inhalten
+        </li>
+        <li>
+            Designentwicklung und Anpassungen
+        </li>
+    </ul>
```

Comment:
Не зовсім правильно, треба обгорнути `<ul>` в `<li>`. Новий список виступає для першого списку як елемент тому він має бути в `<li>`.
```suggestion
  <li>
      <ul>
          <li>
              Erstellung einer Website im Baukasten-Modus
          </li>
          <li>
              Arbeit mit Zahlungssystemen
          </li>
          <li>
              Moderation von Nutzern
          </li>
          <li>
              Einrichtung automatischer E-Mail-Verteiler
          </li>
          <li>
              Organisation und technische Betreuung von Online-Übertragungen
          </li>
          <li>
              Bereitstellung von Multimedia-Inhalten
          </li>
          <li>
              Designentwicklung und Anpassungen
          </li>
      </ul>
    </li>
```